saleae/version.py

```diff
@@ -1 +1 @@
-__version__ = '0.3.0'
+__version__ = '0.4.0'
```

Comment:
So far, I've been bumping the version in a dedicated commit with a summary of changes. I'd like to keep that going. Happy to push a version once this is in, but not in the lump commit.